Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:8.10-alpine
+
+ARG AWS_CLI_VERSION=1.16.86
+
+RUN apk --update add python py-pip bash make imagemagick && \
+    pip --no-cache-dir install awscli==$AWS_CLI_VERSION
```

Comment:
Why do you use `python` ?
This package uses [claudia.js](https://claudiajs.com/) for deploying lambda function, and all dependencies will nr installed via `npm` package manager.